stable_baselines3/common/preprocessing.py

```diff
@@ -81,9 +87,12 @@ def get_obs_shape(observation_space: spaces.Space) -> Tuple[int, ...]:
         return observation_space.shape
     elif isinstance(observation_space, spaces.Discrete):
         # Observation is an int
-        return 1,
+        return 1
+    elif isinstance(observation_space, spaces.MultiDiscrete):
```

Comment:
this is for another issue, no?

Reply:
but we can solve both in that PR, you decide ;)

Reply:
Are these outputs the expected ones?

obs shape: (5,)     obs input: MultiBinary(5)
action shape 5     action input: MultiBinary(5)
obs shape: (1,)     obs input: Discrete(4)
action shape 1     action input: Discrete(4)
obs shape: (3,)    obs input: MultiDiscrete([4 3 2])
action shape 3    action input: MultiDiscrete([4 3 2])

Reply:
Looks good ;)

This only issue with MultiDiscrete is that Gym allows more complicated shapes... We should either throw an error in that case or pre-process a bit.

Reply:
forget about my last comment... it is true for Box, not for `MultiDiscrete`